engine/src/witness/btc/deposits.rs

```diff
@@ -215,10 +215,6 @@ pub fn egress_witnessing(
 				if hash == tx_hash {
 					return Some(TransactionConfirmation {
 						tx_out_id: hash,
-						// we don't care about this value for btc (we don't refund out aggkey)
```

Comment:
@marcellorigotti  - if you look at `record_liability` we don't even use the address that is passed through for Bitcoin, so we can ignore it and submit this. So deleted the comment.